Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
I think the real value in FocusedObjectMessageSender is in it's place alongside the SelectedObjectMessageSender. And the value of both is that they provide a clear paradigm for dealing with the issue of dynamic objects, gaze, and the immutable nature of the KeywordManager. I really suspect that like @dbouianov others will have the same issue and I do like that the toolkit takes a stance on how to solve this.

Ultimately though this goes back to how we're treating this project. Seems like there's a mix of focus on creating functionality for developers to leverage and focus on instructing or educating developers on how they might do something. This falls into the latter.